assets/stylesheets/landing.css

```diff
@@ -32,25 +44,37 @@ body {
 }
 
 .glitch {
-  font-size: 18vh;
+  color: #f8f8f8;
+  font-size: 6.5rem;
+  font-family: 'octopost';
+  text-shadow: 0px 8px 13px rgba(0, 0, 0, 0.2),
+  0px 18px 23px rgba(0, 0, 0, 0.3);
 }
 
 .main-button {
-  position: fixed;
-  top: 90vh;
+  position: absolute;
+  top: 38rem;
```

Comment:
I'd recommend use smaller value, but from bottom, 

`bottom: 12rem` is better than `top:40rem`

Reply:
I'm using a hover effect on that using `margin-top`, and after using `bttom` its not working.

Reply:
@rumansaleem bhaiya ek baar iska bta do